Rakefile

```diff
@@ -37,7 +37,8 @@ task :preview do
     "destination"   => base.join('test/_site').to_s,
     "force_polling" => false,
     "serving"       => true,
-    "theme"         => "minimal-mistakes-jekyll"
+    "theme"         => "minimal-mistakes-jekyll",
+    "port"          => 4001,
```

Comment:
This probably wasn't intended to be committed, right?

Reply:
No, this is very much intentional. It solves the NoMachine problem I describe above, and causes no harm to anyone not using NoMachine.